libnavui-maneuver/api/current.txt

```diff
@@ -974,19 +1004,19 @@ package com.mapbox.navigation.ui.maneuver.view {
     method @Deprecated public void renderSubManeuver(com.mapbox.navigation.ui.maneuver.model.SubManeuver? sub, com.mapbox.navigation.ui.maneuver.model.RoadShield? roadShield = null);
     method @Deprecated public void renderSubManeuver(com.mapbox.navigation.ui.maneuver.model.SubManeuver? sub);
     method public void setUpcomingManeuverRenderingEnabled(boolean value);
-    method public void updateLaneGuidanceIconStyle(@StyleRes int style);
+    method @Deprecated public void updateLaneGuidanceIconStyle(@StyleRes int style);
     method public void updateManeuverViewOptions(com.mapbox.navigation.ui.maneuver.model.ManeuverViewOptions options);
     method @Deprecated public void updatePrimaryManeuverTextAppearance(@StyleRes int style);
     method public void updatePrimaryManeuverTextVisibility(int visibility);
     method @Deprecated public void updateSecondaryManeuverTextAppearance(@StyleRes int style);
     method public void updateSecondaryManeuverVisibility(int visibility);
-    method public void updateStepDistanceTextAppearance(@StyleRes int style);
-    method public void updateStyle(@StyleRes int style);
+    method @Deprecated public void updateStepDistanceTextAppearance(@StyleRes int style);
```

Comment:
creating new options and a new builder to contain a single text seems and adding `@Deprecated`. It seems like a user would actually prefer to use this deprecated function so they don't need to create another builder 🤔 

Is there a functional limitation that is making us do this?

Reply:
The builder was created because there was a limitation with the existing individual functions to change the exit text at runtime. With this change that happened a couple months back, it only supported a few styleable properties which made it inconsistent. This PR adds the missing styleable properties to this builder.

Reply:
oh right, `updateOptions` 👍 . much better.